src/components/Collapse/CollapseFAQ/index.tsx

```
@@ -14,9 +14,7 @@ const CollapseFAQ: React.FC = () => {
       try {
         const categoriesResponse = await getAllFAQCategories()
         setCategories(Array.isArray(categoriesResponse.data.response) ? categoriesResponse.data.response : [])
-      } catch (error) {
-        message.error('Failed to fetch data')
-      }
+      } catch (error) {}
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)